hack/test-templates.sh

```diff
@@ -81,6 +74,21 @@ if limactl ls -q | grep -q "$NAME"; then
 	exit 1
 fi
 
+# Create ${NAME}-tmp to inspect the enabled features.
+# TODO: skip downloading and converting the image here?
```

Comment:
Why? We are about to create an instance for real right afterwards anyways, so it's not like we don't need the image.

Reply:
Probably we should have `limactl create --dry-run-output=<YAML or JSON file path>`, but it is beyond the scope of this PR

Reply:
When/If I get around to implement the `basedOn` mechanism, I was planning to add a `limactl template` command that would output the combined template. It might make sense to add the `--set` and related commands to it as well. But that is now a hypothetical depending on a hypothetical... 😄 

The name was inspired by `helm template`.

Reply:
> This instance is removed in a second without starting it, so it doesn't need the image to be present here.

Yes, but you will create another instance based on the same image again afterwards. So all you would save is the local copy / conversion. I think it is not worth worrying about for CI.

Reply:
(The conversion is not cached though)
- https://github.com/lima-vm/lima/issues/2580